code/game/objects/structures/spawner.dm

```diff
@@ -26,6 +26,7 @@
 
 /obj/structure/spawner/syndicate
 	name = "warp beacon"
+	desc = "A warp beacon used to spawn Syndicate members."
```

Comment:
Spawn is an ooc term, use teleport in or something.

Reply:
Will do. Thank you.